Vagrantfile

```diff
@@ -22,22 +22,22 @@ Vagrant.configure(2) do |config|
     winw1.vm.host_name = "winw1"
     winw1.vm.box = "StefanScherer/windows_2019"  
     winw1.vm.provider :virtualbox do |vb|
-      vb.memory = 8192 # 8GB memory is the normal min for windows clusters
-      vb.cpus = 4 # 2 cores will be flakey on the join and RDP performance
-      vb.gui = true
+      vb.memory = 8192
+      vb.cpus = 4
+      # use rdp to access a GUI if you need it !
+      vb.gui = false
     end
     winw1.vm.network :private_network, ip:"10.20.30.11"
     winw1.vm.synced_folder "./sync", "c:\\sync"
 
     ### for Containerd support
-    #winw1.vm.provision "shell", path: "sync/hyperv.ps1", privileged: true
-    #winw1.vm.provision :reload
-    #winw1.vm.provision "shell", path: "sync/containerd1.ps1", privileged: true
-    #winw1.vm.provision :reload
-    #winw1.vm.provision "shell", path: "sync/containerd2.ps1", privileged: true
+    winw1.vm.provision "shell", path: "sync/hyperv.ps1", privileged: true
+
+    winw1.vm.provision "shell", path: "sync/containerd1.ps1", privileged: true
+    winw1.vm.provision "shell", path: "sync/containerd2.ps1", privileged: true
```

Comment:
We can refactor to run the original script of installContainerD.ps1

Reply:
Hey... is this important ? hoping to demo tomorrow Can we just merge and iterate :)?